qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User Based Login
+
+@selenium
+@defaultLogin
+Scenario: User Based login verification
+   When user verifies modules available on home page
```

Comment:
the above verification is done by which user?

Reply:
It is verified by default admin. which can be verified by ` username= admin, password=admin123`

Reply:
how can one tell that from the gherkin step that is the admin(supper user) who is verifying ?

Reply:
I knew this question will reflect :smile: . Ok

Reply:
can one read the gherkin step and is able to know that the verification is done by the super user(admin)?

Reply:
Thats is another answer given my broad explanation above. thats cool too.

Reply:
can one read the gherkin step and is able to know that the verification is done by the super user(admin)?

Reply:
Yes please.As long as your gherkin scenario is more readable and understandable and is doing the right thing

Reply:
take a look at other steps and compare with the one above!

Reply:
@kdaud i dont see any problem, How would you rename that gherkin according to you.Any suggestion.

Reply:
```js
When user verifies modules available on home page

```
compare with 
```js
And user verifies modules available on homepage after login as clerk
```

Reply:
> ```js
> login as clerk
> ```

Thanks its after login as super User

Reply:
which user is logged in here?
```js
When user verifies modules available on home page
```

Reply:
you need to probably re-read my comment above that explains the workflow and functionalities of the first scenario. this is called a super user . Have updated the scenario to make it more meaning.